.travis.yml

```diff
@@ -1,7 +1,8 @@
 language: go
 
 go:
-  - 1.8.x
```

Comment:
Any specific reason you removed `1.8.x`?

Reply:
1.8 does not work with the underlying library